sdk/core/AzureCore/Source/Util/ApplicationUtil.swift

```diff
@@ -45,7 +47,8 @@ public class ApplicationUtil {
     }
 
     /// Returns the current `UIViewController` for a parent controller.
-    /// - Parameter parent: The parent `UIViewController`. If none provided, will attempt to discover the most relevant controller.
+    /// - Parameter parent: The parent `UIViewController`. If none provided, will attempt to discover the most
+    /// relevant controller.
```

Comment:
Nit - this should be indented to line up after the colon.

Reply:
My inclination would be to leave it. For Xcode, the following all generate the same help strings.

```
/// - Parameter foo: Lines `UIViewController`. If none provided, will attempt to discover the most
/// continues
```

```
/// - Parameter foo: Lines `UIViewController`. If none provided, will attempt to discover the most
///   continues
```

```
/// - Parameter foo: Lines `UIViewController`. If none provided, will attempt to discover the most
///                  continues
```

I'm fine with any option, but the first option is the easiest and thus would be simplest to stay consistent unless we had some kind of swiftlint rule that would autoformat the docstrings.